integration/tests/forseti_integration_testing/controls/inventory.rb

```diff
@@ -0,0 +1,426 @@
+require 'json'
+control 'inventory' do
+
+     describe "Create an inventory" do
+        subject do
+            command("forseti inventory list")
+        end
+
+        before do
+            command("forseti inventory create").result
+        end
+
+        its("stdout") { should match /SUCCESS/}
+
+        after do
+            command("forseti inventory purge 0").result
+        end
+    end
+
+    describe "Create an inventory and check database" do
+        subject do
+            command("mysql -u root --host 127.0.0.1 --database forseti_security --execute \"select count(DISTINCT resource_id) from gcp_inventory where category='resource' and resource_type = 'project';\"")
+        end
+
+        before(:context) do
+            command("forseti inventory create").result
+            command("sudo apt-get -y install mysql-client").result
```

Comment:
This setup seems like it should not be part of a test.  Can it be moved elsewhere?

Reply:
This is required to SSH into the database to check the count of the resources in the org.

Reply:
Moved `mysql-client` installation to the top so it can be used by all the tests.